src/phylum/ci/ci_azure.py

```diff
@@ -112,7 +113,7 @@ def common_lockfile_ancestor_commit(self) -> Optional[str]:
         if pr_tgt_branch.startswith(old_ref_prefix):
             pr_tgt_branch = pr_tgt_branch.replace(old_ref_prefix, new_ref_prefix, 1)
 
-        cmd = f"git merge-base {pr_src_branch} {pr_tgt_branch}".split()
+        cmd = split(f"git merge-base {quote(pr_src_branch)} {quote(pr_tgt_branch)}")
```

Comment:
Personally, all of these function calls and formatting seem like a lot of extra work compared to just using a list in the first place...

```suggestion
        cmd = ["git", "merge-base", pr_src_branch, pr_tgt_branch]
```

Reply:
Good point. It appears the tradeoff in readability has now been surpassed by the use of quoting/sanitization mechanisms and that balance may be better reached with _native_ lists (vs. generated ones) everywhere a command line is needed for a `subprocess.run`. Testing an a re-write is likely coming...

Reply:
re-write complete.